tests/backup/backup_upgrade_test.go

```diff
@@ -0,0 +1,187 @@
+package tests
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	"github.com/pborman/uuid"
+	api "github.com/portworx/px-backup-api/pkg/apis/v1"
+	"github.com/portworx/torpedo/drivers/backup"
+	"github.com/portworx/torpedo/drivers/scheduler"
+	"github.com/portworx/torpedo/pkg/log"
+	. "github.com/portworx/torpedo/tests"
+)
+
+// StorkUpgradeWithBackup validates backups with stork upgrade
+var _ = Describe("{StorkUpgradeWithBackup}", func() {
+	var (
+		contexts             []*scheduler.Context
+		appContexts          []*scheduler.Context
+		backupLocationName   string
+		backupLocationUID    string
+		cloudCredUID         string
+		bkpNamespaces        []string
+		scheduleNames        []string
+		cloudAccountName     string
+		backupName           string
+		scheduleName         string
+		schBackupName        string
+		schPolicyUid         string
+		upgradeStorkImageStr string
+		clusterUid           string
+		clusterStatus        api.ClusterInfo_StatusInfo_Status
+	)
+	var testrailID = 58023 // testrailID corresponds to: https://portworx.testrail.net/index.php?/cases/view/58023
+	labelSelectors := make(map[string]string)
+	cloudCredUIDMap := make(map[string]string)
+	backupLocationMap := make(map[string]string)
+	bkpNamespaces = make([]string, 0)
+	timeStamp := strconv.Itoa(int(time.Now().Unix()))
+	periodicPolicyName := fmt.Sprintf("%s-%s", "periodic", timeStamp)
+	upgradeStorkImageStr = getEnv(upgradeStorkImage, "openstorage/stork:23.2.0")
```

Comment:
move this function call to It

Reply:
It is recommended to separate variable declarations and assignments. You can declare a variable in var() and assign its value in JustBeforeEach.